liblangutil/Exceptions.h

```diff
@@ -44,6 +44,7 @@ struct InternalCompilerError: virtual util::Exception {};
 struct FatalError: virtual util::Exception {};
 struct UnimplementedFeatureError: virtual util::Exception {};
 struct InvalidAstError: virtual util::Exception {};
+struct CommandLineValidationError: virtual util::Exception {};
```

Comment:
I'd create a new `Exceptions.h` file in `solc/` and put the exception there. `liblangutil` is for much more generic.